naming/src/main/java/com/alibaba/nacos/naming/controllers/ServiceController.java

```diff
@@ -364,26 +369,34 @@ public ObjectNode subscribers(HttpServletRequest request) {
         }
     }
     
+    /**
+     * Get all {@link Selector} types.
+     *
+     * @return {@link Selector} types.
+     */
+    @GetMapping("/selector/types")
+    @Secured(parser = NamingResourceParser.class, action = ActionTypes.READ)
```

Comment:
If add secured with `NamingResourceParser`, it will parse request can get namespace, group and serviceName.

But I see the api that has no request, might cause the api always no permission.

Reply:
I think I can remove the @Secured due to this API is common api for all users, it just lists the type of selector for use choosing.